### PR TITLE
feat(Forms): add `toolbarVariant="minimumOneItem"` to Iterate.Toolbar for hiding buttons when there is only one item in the array

### PR DESCRIPTION
**Pull request by tujoworker**
Based on #3877

Quick example using the new Toolbar variant:

```tsx
<Form.Handler>
  <Iterate.Array path="/countries" defaultValue={[null]}>
    <Iterate.ViewContainer toolbarVariant="minimumOneItem"
    >
      <Value.SelectCountry
        label="Land du er statsborger i"
        itemPath="/"
      />
    </Iterate.ViewContainer>

    <Iterate.EditContainer toolbarVariant="minimumOneItem"
    >
      <Field.SelectCountry
        label="Land du er statsborger i"
        itemPath="/"
        required
      />
    </Iterate.EditContainer>
  </Iterate.Array>

  <Iterate.PushButton
    path="/countries"
    pushValue={null}
    text="Legg til flere statsborgerskap"
  />
</Form.Handler>
```

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/EditContainer/info.mdx [hunk 1]
```diff
@@ -47,6 +47,27 @@ render(
 )
 ```
 
+## Get the internal item object
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/EditContainer/info.mdx:50]: This section has just been moved upwards.

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx [hunk 2]
```diff
@@ -51,9 +51,9 @@ With an optional `title` and [Iterate.Toolbar](/uilib/extensions/forms/Iterate/T
 
 ### Initially open
 
-This example uses a customized [Iterate.Toolbar](/uilib/extensions/forms/Iterate/Toolbar/).
+This example uses the container `toolbarVariant` prop with `minimumOneItem`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx:54]:
```suggestion
This example uses the container's `toolbarVariant` prop with `minimumOneItem`.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx [hunk 3]
```diff
@@ -51,9 +51,9 @@ With an optional `title` and [Iterate.Toolbar](/uilib/extensions/forms/Iterate/T
...
 
-It hides the toolbar in the `EditContainer` when there is only one item in the array. And it hides the remove button in the `ViewContainer` when there is only one item in the array.
+It hides the toolbar from the `EditContainer` when there is only one item in the array. And it hides the remove button from the `ViewContainer` when there is only one item in the array.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx:56]:
```suggestion
It hides the toolbar in the `EditContainer` when there is only one item in the array. And it hides the remove button in the `ViewContainer` when there is only one item in the array.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/changelog.mdx [hunk 4]
```diff
@@ -15,7 +15,8 @@ Change log for the Eufemia Forms extension.
 
 ## v10.48
 
-- Make [Iterate.Toolbar](/uilib/extensions/forms/Iterate/Toolbar/) customizable.
+- Make [Iterate.Toolbar](/uilib/extensions/forms/Iterate/Toolbar/) customizable when used inside [Iterate.Array](/uilib/extensions/forms/Iterate/Array/).
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/changelog.mdx:18]:
```suggestion
- Make [Iterate.Toolbar](/uilib/extensions/forms/Iterate/Toolbar/) customizable.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/changelog.mdx [hunk 5]
```diff
@@ -15,7 +15,8 @@ Change log for the Eufemia Forms extension.
...
+- Add new `toolbarVariant` to [Iterate.ViewContainer](/uilib/extensions/forms/Iterate/ViewContainer/) and [Iterate.EditContainer](/uilib/extensions/forms/Iterate/EditContainer/) for hiding the buttons when there is only one item in the array.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/changelog.mdx:19]:
```suggestion
- Add new property `toolbarVariant` to [Iterate.ViewContainer](/uilib/extensions/forms/Iterate/ViewContainer/) and [Iterate.EditContainer](/uilib/extensions/forms/Iterate/EditContainer/) for hiding toolbar buttons when there is only one item in the array.
```

**Review by langz — APPROVED** [review]
Great great 👏 

I'll add a demo using the new toolbar prop, as I like having something visual to look at(to better understand) than just [reading](https://eufemia-git-feat-iterate-toolbar-variant-eufemia.vercel.app/uilib/extensions/forms/Iterate/ViewContainer/info/#variants).

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx [hunk 6]
```diff
@@ -51,7 +51,7 @@ With an optional `title` and [Iterate.Toolbar](/uilib/extensions/forms/Iterate/T
 
 ### Initially open
 
-This example uses a customized [Iterate.Toolbar](/uilib/extensions/forms/Iterate/Toolbar/).
+This example uses the container's `toolbarVariant` prop with `minimumOneItem`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/demos.mdx:54]:
```suggestion
This example uses the container's `toolbarVariant` property with the value `minimumOneItem`.
```

**Comment by langz** [comment]
I have one more thought about this demo/example.
Should it be possible to remove your selection? Or perhaps the whole point is that this field(statsborgerskap) is required?
If it's not required(optional), then I would expect being able to remove my selection:


https://github.com/user-attachments/assets/4e0e62fc-e60b-4eba-aae3-cc3e674207aa

**Comment by tujoworker** [comment]
> Should it be possible to remove your selection?

Not in this case. So yeah, then you would not need the toolbarVariant.

**Review by langz — APPROVED** [review]

**Pull request merged**